### PR TITLE
fix(docs): Update the repository path url to a valid value

### PR DESCRIPTION
**Pull request by revitalbarletz**
The path values should be either helm-guestbook or kustomize-guestbook instead of just guestbook. Fixed the tutorial accordignly as well as updated the image to include the right values.

Issue: https://github.com/argoproj/argo-cd/issues/20822. 

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] The title of the PR conforms to the [Toolchain Guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/toolchain-guide/#title-of-the-pr)
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/blob/master/community/CONTRIBUTING.md#legal)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).
* [ ] My new feature complies with the [feature status](https://github.com/argoproj/argoproj/blob/master/community/feature-status.md) guidelines.
* [ ] I have added a brief description of why this PR is necessary and/or what this PR solves.
* [ ] Optional. My organization is added to USERS.md.
* [ ] Optional. For bug fixes, I've indicated what older releases this fix should be cherry-picked into (this may or may not happen depending on risk/complexity).

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ Give your app the name `guestbook`, use the project `default`, and leave the syn
 
 ![app information](assets/app-ui-information.png)
 
-Connect the [https://github.com/argoproj/argocd-example-apps.git](https://github.com/argoproj/argocd-example-apps.git) repo to Argo CD by setting repository url to the github repo url, leave revision as `HEAD`, and set the path to `guestbook`:
+Connect the [https://github.com/argoproj/argocd-example-apps.git](https://github.com/argoproj/argocd-example-apps.git) repo to Argo CD by setting repository url to the github repo url, leave revision as `HEAD`, and set the path to either `helm-guestbook` or `kustomize-guestbook`:
```
Comment (nitishfy) [inline docs/getting_started.md:173]:
```suggestion
Connect the [https://github.com/argoproj/argocd-example-apps.git](https://github.com/argoproj/argocd-example-apps.git) repo to Argo CD by setting repository url to the github repo url, leave revision as `HEAD`, and set the path to either `helm-guestbook` or `kustomize-guestbook`:
```

why set to `helm-guestbook` or `kustomize-guestbook`? The [guestbook](https://github.com/argoproj/argocd-example-apps/tree/master/guestbook) path exists in the repo url already.

**Review by nitishfy — COMMENTED** [review]
PTAL

File: docs/getting_started.md (hunk 1 above)
Comment (revitalbarletz) [reply docs/getting_started.md:173]: When I set to guestbook only it does not work. Does it work for you?

Comment (revitalbarletz) [reply docs/getting_started.md:173]: It does not work with only guestbook. I am not able proceeding and creating the app.

**Comment by revitalbarletz** [comment]
It does not work with guestbook, I am not able to create the application.

**Comment by blakepettersson** [comment]
@revitalbarletz this manifest works for me using `guestbook`

```yaml
apiVersion: argoproj.io/v1alpha1
kind: Application
metadata:
  name: guestbook
  namespace: argocd
spec:
  destination:
    namespace: guestbook
    server: 'https://kubernetes.default.svc'
  sources:
  - path: guestbook
    repoURL: 'https://github.com/argoproj/argocd-example-apps'
  project: default
  syncPolicy:
    automated:
      prune: true
    syncOptions:
      - CreateNamespace=true
      - PrunePropagationPolicy=foreground
      - PruneLast=true
```

**Comment by nitishfy** [comment]
For me it works with guestbook too.

```
project: default
source:
  repoURL: https://github.com/argoproj/argocd-example-apps
  path: guestbook
  targetRevision: HEAD
destination:
  server: https://kubernetes.default.svc
  namespace: default
```

If it doesn't show you the `guestbook` on the UI, try inserting the path manually instead.
image

**Comment by revitalbarletz** [comment]
Thank you @nitishfy for checking this out. I Will debug my failure and close this PR.

**Pull request closed without merging**